richie/settings.py

```diff
@@ -13,6 +15,39 @@
 DATA_DIR = os.path.join("/", "data")
 
 
+def get_release():
+    """Get current release of the application
+
+    By release, we mean the latest commit's sha1. Two strategies are tested to
+    get this info:
+
+    1. from the version.json file à la Mozilla [1] (if any)
+    2. from the git repository (if any)
+
+    If none of those returns the release, it defaults to "NA".
+
+    [1] https://github.com/mozilla-services/Dockerflow/blob/master/docs/version_object.md
+    """
+    project_dir = os.path.abspath(os.path.join(BASE_DIR, os.pardir))
+
+    # Try to get the current release from the version.json file generated by the
+    # CI during the Docker image build
+    try:
+        with open(os.path.join(project_dir, "version.json")) as version:
+            return json.load(version)["commit"]
+    except FileNotFoundError:
+        pass
+
+    # Get latest commit sha1 from the local git repository
+    try:
+        return raven.fetch_git_sha(project_dir)
+    except raven.exceptions.InvalidGitRepository:
+        pass
+
+    # Default: not available
+    return "NA"
```

Comment:
I know this is just a failsafe, but maybe we could add datetime info here, just in case we start seeing these it might help us to pinpoint their source?

(of course we could also try more complicated stuff like detect the OS etc. but I'm thinking datetime is trivial to add and could prove helpful)

Reply:
Contextual informations are already collected and sent by Sentry/raven (OS, user, server hostname, ip, etc.). I think those metadata would be sufficient to track who's sending sentry events from it's laptop :stuck_out_tongue:

Reply:
Alright then, `"NA"` is good enough!!